packages/react-expandable-table/src/styles.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (c) Quid, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @flow
+import * as React from 'react';
+import styled from '@emotion/styled/macro';
+import { GroupedVirtuoso } from 'react-virtuoso';
+import { themes, textStyles } from '@quid/theme';
+import { Icon, Text } from '@quid/react-core';
+import { Container } from '@quid/react-tooltip';
+import isPropValid from '@emotion/is-prop-valid';
+
+const alignToFlex = (align = 'left') =>
+  align === 'left' ? 'flex-start' : align === 'right' ? 'flex-end' : 'center';
```

Comment:
Hard to follow.

Reply:
What's unclear? This is mapping the `align` property to the corresponding flex alignment value.

Reply:
I agree here with @pdw207.

Reply:
And I repeat my question 😛 What's unclear?

Reply:
Multiple ternary operators in a row is hard to follow.

Reply:
I agree, a ternary operator seems good to make an if in a line. start nesting ternary operators is hard to follow. 
Please let's convert to if statements.
Let's optimize for readability than line counts.